spec/hack/build-and-push_spec.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+eval "$(shellspec - -c) exit 1"
+
+check_lines() {
+  while read -r line; do
+    if [[ "$line" == quay.io/* ]] && [[ ! "$line" =~ ([0-9]+\.[0-9]-1234)?@sha256:5678$ ]]; then
```

Comment:
Nitpick: I'd be a bit more strict here (regex is most likely wrong, typing from the top of my head...

```suggestion
    if [[ "$line" == quay.io/* ]] && [[ ! "$line" =~ ^quay.io/(?:[a-z\-/]+){2}:[0-9]+\.[0-9]-1234@sha256:5678$ ]]; then
```

Reply:
done